src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -992,6 +995,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
 					Command = commandsViewModel.UnpinItemFromStartCommand,
 					ShowOnShift = true,
 					ShowItem = selectedItems.All(x => !x.IsShortcut && (x.PrimaryItemAttribute == StorageItemTypes.Folder || x.IsExecutable) && !x.IsArchive && x.IsItemPinnedToStart),
+					ShowInRecycleBin = true,
```

Comment:
I don't think we want this one, this is about items contained in the recycle bin not the recycle bin folder.

Reply:
@Hark64 can you open a PR to resolve?

Reply:
Sorry about that. I opened PR #10945 to address this.